webapp/lib/app/configurator/standard_messages/controller.dart

```diff
@@ -202,8 +202,7 @@ class MessagesConfiguratorController extends ConfiguratorController {
       await Future.wait([platform.updateSuggestedReplies(diffData.editedMessages), platform.deleteSuggestedReplies(diffData.deletedMessages)]);
       _view
         ..unsavedChanges = false
-        ..showSaveStatus('Saved!', autoHide: true)
-        ..clearUnsavedIndicators();
+        ..showSaveStatus('Published for saving!', autoHide: true);
```

Comment:
Can we roll this back to "Saved!"? (even though we know it's not quite true)

"published for saving" won't make sense as a description to the users

Reply:
Fixed!